cfgov/v1/atomic_elements/molecules.py

```diff
@@ -116,43 +116,74 @@ class Meta:
 class Hero(blocks.StructBlock):
     heading = blocks.CharBlock(
         required=False,
-        help_text='Maximum character count: 25 (including spaces)'
+        help_text=mark_safe(
+            'For complete guidelines on creating heroes, visit our '
+            '<a href="https://cfpb.github.io/design-manual/global-elements/heroes.html">'  # noqa: E501
+            'Design Manual</a>.'
+            '<ul class="help">Character counts (including spaces) at largest '
+            'breakpoint:'
+            '<li>&bull; 41 characters max (one-line heading)</li>'
+            '<li>&bull; 82 characters max (two-line heading)</li></ul>')
     )
     body = blocks.RichTextBlock(
+        label="Sub-heading",
         required=False,
-        help_text='Maximum character count: 185 (including spaces)'
+        help_text=mark_safe(
+            '<ul class="help">Character counts (including spaces) at largest '
+            'breakpoint:'
+            '<li>&bull; 165-186 characters (after a one-line heading)</li>'
+            '<li>&bull; 108-124 characters (after a two-line heading)</li>'
+            '</ul>')
     )
-    background_color = blocks.CharBlock(
+    image = ImageChooserBlock(
+        label="Large image",
         required=False,
-        help_text='Specify a hex value (with the # sign) '
-                  'from our official palette: '
-                  'https://github.com/cfpb/cf-theme-cfpb/blob/'
-                  'master/src/color-palette.less'
+        help_text=mark_safe(
+            'When saving illustrations, use a transparent background. '
+            '<a href="https://cfpb.github.io/design-manual/global-elements/heroes.html#style">'  # noqa: E501
+            'See image dimension guidelines.</a>')
     )
-    is_white_text = blocks.BooleanBlock(
+    small_image = ImageChooserBlock(
         required=False,
-        help_text='Turns the hero text white. Useful if using '
-                  'a dark background color or background image.'
+        help_text=mark_safe(
+            '<b>Optional.</b> No small image required for the '
```

Comment:
We just decided on a small update here to add a little more context for the small image. The helper text should now be `<b>Optional.</b> Provides an alternate image for small displays when using a photo or bleeding hero. Not required for the standard illustration. <a href="https://cfpb.github.io/design-manual/global-elements/heroes.html#style">See image dimension guidelines.</a>`.

Reply:
Got it. I've updated the PR to use that language, and the screenshot above now reflects that.

Reply:
(Oops, I had the wrong image in there for a bit. Updated now.)

Reply:
Looks good to me, thanks!